gc/stats/ConcurrentPhaseStatsBase.hpp

```diff
@@ -57,6 +61,8 @@ class MM_ConcurrentPhaseStatsBase : public MM_Base
 private:
 protected:
 public:
+	virtual void handleConcurrentEndInternal(J9HookInterface** hook, uintptr_t eventNum, void* eventData, MM_VerboseHandlerOutputStandard* verbose) {};
```

Comment:
Not sure all parameters are required (`J9HookInterface** hook, uintptr_t eventNum`). Looks other methods have these redundant variables as well.

Reply:
Having an handler method in a stats class like this is "awkward", these belong in the verbose handler class. I see the intent though and what you're trying to do.

Reply:
Let's do something similar to the initial idea with an if branch in the handler, see my comments in `MM_VerboseHandlerOutputStandard::handleConcurrentEndInternal`